### DISCUSSION TITLE
Welcome to cra-template-dapp Discussions!

### DISCUSSION THREAD

**Discussion by huseyindeniz** [huseyindeniz/cra-template-dapp#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clemrider** [comment]
Hi, after several week ends on my spare time, I finally get a react app that allow me to connect to metamask and to my own smart contract. Before going further, by building a simple web3 game, I thought how to navigate in my app easy ? I was a bit bored to create app each  time, and install dep manualy, that's why I'm trying your template. it looks nice, and maybe after I resolve the first errors, I'll enjoy it and build my app =)

**Reply by huseyindeniz** [reply]
hi @clemrider thank you for trying my template. you might find this full example -including hardhat project- useful => [simple-dapp](https://github.com/huseyindeniz/simple-dapp) and please let me know if you need help. (Documentation is still in progress)

**Reply by clemrider** [reply]
My goal atm is to build a dapp, run it locally, but connected on the testnet polygon mumbai. There is so many differents way to work, I choosed this one to dev my app, my game. I've enough friends and connexions to launch a beta test on mumbai. So for the moment, I just need to keep learning react, solidity.

**Reply by huseyindeniz** [reply]
quick info about network configuration: you can set supported networks and default network by changing **SUPPORTED_NETWORKS** and **DEFAULT_NETWORK** constants which are defined in the **src/features/wallet/config.ts** file.

**Comment by clemrider** [comment]
Hi,Like I said, im working on a dapp on m'y spare time, si i'd like to speak with you directly. I will try your templates asap, it will help me to understand how some dep works. Maybe we can work on m'y project together, i read that you looking for projects  to get involved in. Let me know what you think about it 😉 have a nice day, see you soon

**Reply by huseyindeniz** [reply]
hi and sorry, I was on holiday and just returned back. I checked your profile but couldn't find any email. You can send me an email and we can discuss your project details.

**Comment by clemrider** [comment]
Hi, here m'y email : wg.enterprise.fr@gmail.com
I wrote you on Messenger 😁
